Description: accept older branch of crate reqwest
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2024-09-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,7 +63,7 @@
 rand = "0.8.4"
 regex = "1.10.3"
 reqsign = { version = "0.16.0", optional = true }
-reqwest = { version = "0.12", features = [
+reqwest = { version = ">= 0.11.27, <= 0.12", features = [
   "json",
   "blocking",
   "stream",
